Userland/Services/ChessEngine/ChessEngine.cpp

```diff
@@ -66,5 +66,10 @@ void ChessEngine::handle_go(GoCommand const& command)
 void ChessEngine::handle_quit()
 {
     if (on_quit)
-        on_quit();
```

Comment:
please don't iterate in the same PR. just add status codes to the original quit commit